R/pkg/R/mllib.R

```diff
@@ -1852,9 +1867,9 @@ summary.treeEnsemble <- function(model) {
 
 #  Get the summary of a Random Forest Regression Model
 
-#' @return \code{summary} returns a summary object of the fitted model, a list of components
-#'         including formula, number of features, list of features, feature importances, number of
-#'         trees, and tree weights
+#' @return \code{summary} returns summary information of the fitted model, which is a list.
+#'         The list of components includes \code{ans} (formula, number of features, list of features,
+#'         feature importances, number of trees, and tree weights).
```

Comment:
that's a function call, see https://github.com/apache/spark/pull/16150/files/6039c1f742f77b4ceff624b3979afbaa9e80d77a#diff-7ede1519b4a56647801b51af33c2dd18R1851